composer.json

```diff
@@ -12,12 +12,11 @@
     "require": {
         "php": "^7.2|^8.0",
         "ramsey/uuid": "^3|^4",
-        "symfony/yaml": "^3.2|^4.0|^5.0"
+        "symfony/yaml": "^4.0|^5.0"
     },
     "require-dev": {
-        "phpstan/phpstan": "^0.11.16|^0.12.52",
-        "phpunit/phpunit": "^7.0|^8.0|^9.3",
-        "friendsofphp/php-cs-fixer": "^2.15"
+        "phpstan/phpstan": "^0.12.52",
+        "phpunit/phpunit": "^8.5|^9.4"
```

Comment:
I've updated these two dependencies because the older versions aren't needed anymore for the test suite.